src/test/kotlin/no/nav/syfo/service/SykepengerMaxDateServiceSpek.kt

```diff
@@ -95,5 +98,55 @@ object SykepengerMaxDateServiceSpek : Spek({
             val newStoredDate = embeddedDatabase.fetchSykepengerMaxDateByFnr("123")
             assertEquals(null, newStoredDate)
         }
+
+        it("Should store spleis utbetaling") {
```

Comment:
Kan vi lage egen DAO + test/Spek for spleis-utbetalingene? Jeg synes det er greit å ha en DAO pr databasetabell.